lib/Controller/APIv2Controller.php

```diff
@@ -363,6 +363,7 @@ protected function getPreview(string $owner, int $fileId, string $filePath): arr
 			'isMimeTypeIcon' => true,
 			'fileId' => $fileId,
 			'view' => $info['view'] ?: 'files',
+			'filename' => basename($filePath),
```

Comment:
@nickvergessen This is an REST API change, shouldn't it be documented?

Reply:
@jancborchardt can you add it in https://github.com/nextcloud/activity/blob/master/docs/endpoint-v2.md#preview-element ? :P

Reply:
https://github.com/nextcloud/activity/pull/588